pkg/ccl/benchccl/rttanalysisccl/testdata/benchmark_expectations

```diff
@@ -1,19 +1,19 @@
 exp,benchmark
-17,AlterPrimaryRegion/alter_empty_database_alter_primary_region
-23,AlterPrimaryRegion/alter_empty_database_set_initial_primary_region
-17,AlterPrimaryRegion/alter_populated_database_alter_primary_region
-24,AlterPrimaryRegion/alter_populated_database_set_initial_primary_region
-17,AlterRegions/alter_empty_database_add_region
-17,AlterRegions/alter_empty_database_drop_region
-17,AlterRegions/alter_populated_database_add_region
-17,AlterRegions/alter_populated_database_drop_region
-17,AlterSurvivalGoals/alter_empty_database_from_region_to_zone
-17,AlterSurvivalGoals/alter_empty_database_from_zone_to_region
-37,AlterSurvivalGoals/alter_populated_database_from_region_to_zone
-37,AlterSurvivalGoals/alter_populated_database_from_zone_to_region
-15,AlterTableLocality/alter_from_global_to_rbr
-17,AlterTableLocality/alter_from_global_to_regional_by_table
-14,AlterTableLocality/alter_from_rbr_to_global
-14,AlterTableLocality/alter_from_rbr_to_regional_by_table
-17,AlterTableLocality/alter_from_regional_by_table_to_global
-15,AlterTableLocality/alter_from_regional_by_table_to_rbr
+20,AlterPrimaryRegion/alter_empty_database_alter_primary_region
+25,AlterPrimaryRegion/alter_empty_database_set_initial_primary_region
+20,AlterPrimaryRegion/alter_populated_database_alter_primary_region
+26,AlterPrimaryRegion/alter_populated_database_set_initial_primary_region
+19,AlterRegions/alter_empty_database_add_region
+20,AlterRegions/alter_empty_database_drop_region
+19,AlterRegions/alter_populated_database_add_region
+20,AlterRegions/alter_populated_database_drop_region
+21,AlterSurvivalGoals/alter_empty_database_from_region_to_zone
+21,AlterSurvivalGoals/alter_empty_database_from_zone_to_region
+41,AlterSurvivalGoals/alter_populated_database_from_region_to_zone
+41,AlterSurvivalGoals/alter_populated_database_from_zone_to_region
+23,AlterTableLocality/alter_from_global_to_rbr
+26,AlterTableLocality/alter_from_global_to_regional_by_table
+19,AlterTableLocality/alter_from_rbr_to_global
+19,AlterTableLocality/alter_from_rbr_to_regional_by_table
+26,AlterTableLocality/alter_from_regional_by_table_to_global
+23,AlterTableLocality/alter_from_regional_by_table_to_rbr
```

Comment:
Those are some significant performance regressions. Is this PR really the best that can be done to address the issue at hand?